Airports/EGBJ/SMR/Labels.txt

```diff
@@ -42,6 +42,11 @@
 "H" N051.53.44.290 W002.10.03.455 standHold
 "H" N051.53.44.851 W002.09.55.008 standHold
 "H" N051.53.45.112 W002.09.51.067 standHold
+"H" N051.53.23.521 W002.09.40.925 standHold
+
+;Heli Holds
+"H-X" N051.53.35.822 W002.09.41.507 standHold
```

Comment:
It might just be the particular screenshot, but H-X looks more in the middle between the end of 22 and A3:
![image](https://user-images.githubusercontent.com/14115426/178136857-fa471fe1-1352-4b39-b571-ee41de90dbc8.png)
![image](https://user-images.githubusercontent.com/14115426/178136879-7844d44c-f04b-4aa3-ab28-82011e849c0b.png)

Reply:
Relocated. Lesson learned - don't trust Google Earth!